origami.json

```diff
@@ -22,34 +22,39 @@
     "demos": [
         {
             "name": "header",
+            "title": "Header",
             "data": "demos/src/header.js",
             "template": "demos/src/header.mustache",
             "expanded": true,
             "description": ""
         },
         {
             "name": "mega-menu",
+            "title": "Mega menu",
             "data": "demos/src/mega-menu.js",
             "template": "demos/src/mega-menu.mustache",
             "expanded": false,
             "description": ""
         },
         {
             "name": "subnav",
+            "title": "Header with subnav",
             "data": "demos/src/subnav.js",
             "template": "demos/src/subnav.mustache",
             "expanded": false,
             "description": ""
         },
         {
             "name": "simple-header",
+            "title": "Simple Header",
             "data": "demos/src/header.js",
             "template": "demos/src/simple-header.mustache",
             "expanded": true,
             "description": ""
         },
         {
             "name": "sticky-header",
+            "title": "Sticky Header",
```

Comment:
Can these titles be sentence case (eg Sticky header). We currently have a mix in this file anyway...